test/integration/default/build/ec2.tf

```diff
@@ -0,0 +1,149 @@
+# Contains resources and outputs related to testing the aws_ec2_* resources.
+
+#======================================================#
+#                    EC2 Instances
+#======================================================#
+
+# Test fixture info:
+# instance |    OS   | has_role? | instance_type
+# -----------------------------------------------
+#  alpha   | debian  |    N      |  t2.micro
+#   beta   | centos  |    Y      |  t2.small
+
+
+resource "aws_instance" "alpha" {
+  ami           = "${data.aws_ami.debian.id}"
+  instance_type = "t2.micro"
+
+  tags {
+    Name = "${terraform.env}.alpha"
+    X-Project = "inspec"
+  }
+}
+
+resource "aws_instance" "beta" {
+  ami           = "${data.aws_ami.centos.id}"
+  instance_type = "t2.small"
```

Comment:
Since we don't check type, could we make this a micro too? Not sure if the cost difference matters though.

Reply:
We do in fact test, here: https://github.com/chef/inspec-aws/pull/128/files#diff-a42bfa370ab2f6c44f3b6adcc346d767R64

Reply:
https://giphy.com/gifs/cheezburger-invisible-hide-v5VOE6TRh9JPa/fullscreen